### ISSUE TITLE
No Tasks found for New Work Flows

### ISSUE THREAD

**Issue by Techguyrosh** [AOT-Technologies/forms-flow-ai#1617]
### Please provide information about your setup

- formsflow.ai Version: v5.1.1
- Browser: Google Chrome, Microsoft Edge, Mozilla Firefox
- Browser version: Chrome - 116.0.5845.182, Microsoft Edge - 116.0.1938.76, Firefox - 117.0

### Steps to Reproduce

1. Full server deployment done with consultants (AOT)
2. Creation of 20 unique forms
3. Creation of 2 Default Workflows
4. Association of Forms to Work flows
5. Inclusion of Task Variables
6. Preview, confirm and publish form
7. Test submission
8. Form successfully submitted
9. Application can be viewed, however does not create task nor shows associated task location on Process Diagram
![FORMSFLOWSCREENSHOT](https://github.com/AOT-Technologies/forms-flow-ai/assets/144851681/dbf1adcd-26b6-4ce7-92ad-44598f0ddb55)

### Observed Behaviour

No Tasks being Created after form submission. 
The Form displays in the Applications Window, as the client / creator I can view the form. However when logged in as a reviewer, tasks does not display for the new workflow.

### Expected Behaviour
Tasks to be created for each submission of forms.

**Comment by Techguyrosh** [comment]
Additional Information: The process does show up in the cockpit for the camunda portal but shows 0 for incidents and running instances

**Comment by sumesh-aot** [comment]
@Techguyrosh Could be issue on workflow initialization. Can you see anything on the BPM logs ?

**Comment by Techguyrosh** [comment]
Hi @sumesh-aot is there a way to view logs from the camunda web portal?

**Comment by sumesh-aot** [comment]
@Techguyrosh No, will have to look into the container logs. Can you reach out to your account manager from AOT (Barry), so that team would be able to help you on this ?

**Comment by Techguyrosh** [comment]
Managed to get it working, after checking container logs found that there were variables names being called that were different from what the form is parsing
